gobblin-hive-registration/src/main/java/org/apache/gobblin/hive/orc/HiveOrcSerDeManager.java

```diff
@@ -261,14 +250,11 @@ private void addSchemaProperties(Path path, HiveRegistrationUnit hiveUnit)
    * org.apache.hadoop.hive.serde.serdeConstants#LIST_COLUMNS and
    * org.apache.hadoop.hive.serde.serdeConstants#LIST_COLUMN_TYPES
    *
-   * Keeping {@link #SCHEMA_LITERAL} will be a nice-to-have thing but not actually necessary in terms of functionality.
```

Comment:
Not an issue, but call out on changing the documentation as well, usually we forget about that and leads to code smell.